src/components/SocialIcon/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import Link from '../Link'
+import { ReactComponent as TwitterIcon } from '../../../static/img/community/icon-twitter.svg'
+import { ReactComponent as GithubIcon } from '../../../static/img/community/icon-github.svg'
```

Comment:
I think we should move them out of community if we use them in multiple places ... to be honest I don't even see a problem in certain cases to duplicate them so that we can manage them independently

Reply:
in this case probably just move them to `./`

Reply:
I have no problem with that, the existing icons aren't used in many other places anyway.

Also, with the SVGR resizing fix, we can use imported SVGs instead of static ones with no downsides.
I'll just do the ones involved here for now, but we may want to consider giving most of our static SVG icons this treatment later so they can be run through SVGR and maybe inlined.